README.md

```diff
@@ -38,229 +88,412 @@
-                        to dependencies.
+The typical workflow looks like this:
 
-  shell                 The same as esy build-shell, but creates a "relaxed"
-                        environment - meaning it also inherits your existing
-                        shell.
+0. Install esy:
+    ```
+    % npm install -g esy
+    ```
 
-  release TYPE          Create a release of type TYPE ("dev", "pack" or "bin").
+1. Clone the project:
+    ```
+    % git clone git@github.com:esy-ocaml/esy-ocaml-project.git
+    % cd esy-ocaml-project
+    ```
 
-  print-env             Prints esy environment on stdout.
+2. Install project's dependencies source code:
+    ```
+    % esy install
+    ```
 
-  build-shell           Drops into a shell with environment matching your
-                        package's build environment.
+3. Perform an initial build of the project's dependencies and of the project
+   itself:
+    ```
+    % esy build
+    ```
 
-  build-eject           Creates node_modules/.cache/esy/build-eject/Makefile,
-                        which is later can be used for building without the NodeJS
-                        runtime.
+4. Test the compiled executables inside the project's environment:
+    ```
+    % esy ./_build/default/bin/hello.exe
+    ```
 
-                        Unsupported form: build-eject [cygwin | linux | darwin]
-                        Ejects a build for the specific platform. This
-                        build-eject form is not officially supported and will
-                        be removed soon. It is currently here for debugging
-                        purposes.
+5. Hack on project's source code and rebuild the project:
+    ```
+    % esy build
+    ```
 
+Also:
 
-  import-opam           Read a provided opam file and print esy-enabled
-                        package.json conents on stdout. Example:
+6. It is possible to invoke any command from within the project's sandbox.
+   For example build & run tests with:
+    ```
+    % esy make test
+    ```
+   You can run any command command inside the project environment by just
+   prefixing it with `esy`:
+    ```
+    % esy <anycommand>
+    ```
 
-                          esy import-opam lwt 3.0.0 ./opam
+7. To shell into the project's sandbox:
+    ```
+    % esy shell
+    ```
 
-  <command>             Executes <command> as if you had executed it inside of
-                        esy shell.
+8. For more options:
+    ```
+    % esy help
+    ```
+
+### Configuring Your `package.json`
+
+`esy` knows how to build your package and its dependencies by looking at the
+`esy` config section in your `package.json`.
+
+This is how it looks for a [jbuilder][] based project:
 
 ```
+{
+  "name": "example-package",
+  "version": "1.0.0",
 
-## Try An Example
+  "esy": {
+    "build": [
+      "jbuilder build bin/hello.exe"
+    ],
+    "install": [
+      "jbuilder build @install",
+      "jbuilder install --prefix=$cur__install"
```

Comment:
`esy-installer`  instead of `jbuilder install --prefix=$cur__install`?

Reply:
Good point. I need to fix example projects too.